mentat/code_file_manager.py

```diff
@@ -137,7 +134,7 @@ def _set_file_paths(
             cprint("The following paths do not exist:")
             print("\n".join(invalid_paths))
             print("Exiting...")
-            exit()
+            raise UserError()
```

Comment:
We could change the error to be the following paths don't exist instead of cprinting, and maybe get rid of the 'exiting...'? Since that should be standard (either always have it, or always don't have it).

Reply:
It turns out the warning for bad file paths got broken when we switched to globbing so I moved this chunk of code to expand_paths() in app.py.